WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/handlers/BackupDownloadStatusHandler.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.ui.jetpack.backup.download.handlers
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flowOn
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.ui.jetpack.backup.download.usecases.BackupDownloadStatusUseCase
+import org.wordpress.android.ui.pages.SnackbarMessageHolder
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.viewmodel.Event
+import javax.inject.Inject
+import javax.inject.Named
+
+class BackupDownloadStatusHandler @Inject constructor(
```

Comment:
❓ Same as above - Do we plan to add more logic into this class?

Reply:
See above comment. :) thanks